k8s/deployment.yaml

```
@@ -0,0 +1,39 @@
+kind: Service
+metadata:
+  name: web-service
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 80
+    protocol: TCP
+  selector:
+    app: webapplication
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)